src/components/tutorial-collection-cards/components/card-heading/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { ElementType } from 'react'
+import classNames from 'classnames'
+import { CardHeadingProps } from './types'
+import CompanyLogo from './company-logo'
+import s from './card-heading.module.css'
+
+function CardHeading({ text, level, logo }: CardHeadingProps) {
+  const hasLogo = Boolean(logo)
+  const HeadingElement = `h${level}` as ElementType
+  return (
+    <HeadingElement className={classNames(s.root, { [s.hasLogo]: hasLogo })}>
+      <span className={classNames(hasLogo && 'g-screen-reader-only')}>
+        {text}
+      </span>
+      {hasLogo && (
```

Comment:
If this is shared with the tutorial card, perhaps we write a note that only collection cards should render a logo?

Reply:
Great call... Looking at this again, I think I should probably move the `logo` stuff into `components/collection-card`. Going to do that now - I'll have to add more CSS for logo sizing tweaks (chatted with Brayden via Slack on that), so don't want collection-card-specific stuff to overwhelm these shared elements 👍

Reply:
✅ Done. Moved `logo` stuff into `collection-card`, was a bit of a refactor, but I think it's much clearer now.